tests/test_theme_toronto.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import altair as alt
+
+# import theme_toronto.py file with functions. 
+from hueniversitypy.theme_toronto import *
+
+# creating data to plot for tests
+data = pd.melt(pd.DataFrame({'A': [1, 4, 6, 8, 3, 6, 7], 
+                     'B':[2, 3, 7, 4, 8, 4, 9], 
+                     'C':[3, 6, 3, 5, 8, 2, 8] }))
+
+# call the theme function to enforce themes on all subsequent plots 
+alt.themes.register("theme_toronto", theme_toronto)
+alt.themes.enable("theme_toronto")
+
+# creating plot for testing with theme
+test_plot = alt.Chart(data).mark_point().encode(
+    x='count()',
+    y='value',
+    color='variable',
+).properties(title = 'Test')
+
+def test_chart_object():
+    ''' A function that tests if an altair chart object is created  '''
+    
+    assert str(type(test_plot)) == f"<class 'altair.vegalite.v{alt.__version__[0]}.api.Chart'>"
+    
+def test_colours():
```

Comment:
Same suggestion as above to include `test_plot` as an argument to the function.

Reply:
Will do!